web/src/compose_recipient.js

```diff
@@ -319,6 +314,21 @@ function focus_compose_recipient() {
     $("#compose_recipient_selection_dropdown").trigger("focus");
 }
 
+function on_hidden_callback() {
```

Comment:
Hmm, I worry that this is incorrectly going to happen even if you click outside the popover in some specific place, like the compose textarea.

Reply:
Though I cannot reproduce an issue of this form, hmm.

Reply:
Nice thinking! I looked into this and seems like `tippy` handles this hide event on `mousedown` so it is always triggered before the `click` event on textarea (which is mousedown + mouseup). So, the focus moves from say topic input to compose textarea when user clicks outside the popover.

Since we are setting pretty harmless focus on some input areas, I don't think it will be a hinderence to other event handlers.